src/reducers/clusterReducer.js

```diff
@@ -134,6 +134,10 @@ const clusterReducer = produce((draft, action) => {
     case types.CLUSTER_PATCH_ERROR:
       draft.items[action.cluster.id] = action.cluster;
       return;
+
+    case types.NODEPOOL_CREATE_SUCCESS:
+      draft.items[action.clusterId].push(action.nodePool.id);
```

Comment:
not sure about the API.. does it make sense to push the nodepool into the clusters list?

Reply:
It is actually an error, it has to be pushed inside the `nodePools`array of a cluster. 
Didn't noticed it because endpoint is not working and so this code has never been reached.
Thanks!